api-report/driver-definitions.api.md

```diff
@@ -49,6 +49,32 @@ export enum DriverErrorType {
     writeError = "writeError"
 }
 
+// @public
+export const DriverErrorTypes: {
```

Comment:
Note for reviewers: This object contains all of the error types from `FluidErrorTypes` in `core-interfaces`. A couple of these do not exist on the existing `DriverErrorType`. Namely:
* `dataCorruptionError`
* `dataProcessingError`

Questions: Do these 2 belong here semantically? Should they only exist in `ContainerErrorTypes` and not `FluidErrorTypes`?

Reply:
since we don't need them today, i'd propose we move them. its an open question to me if we would ever want them at this layer, i can see both arguments, and it boils down to if the driver knows about content, or is fully content agnostic.

Reply:
that said, we may want to keep them at the core-interfaces layer, but not propagate to the driver layer, as if we move them, it becomes more complicated where we put the implementations.